### ISSUE TITLE
🐛 Inner comments lost when formatting case-less switch statement

### ISSUE THREAD

**Issue by yassere** [rome/tools#1565]
If a switch statement contains comments but no case clauses, it parses into a `JSSwitchStatement` with no `innerComments`, and formatting it will cause those comments to be lost.

There’s not much use for a case-less switch statement, but if someone comments out the case clauses while testing, those comments shouldn’t disappear when they format-on-save.

## Environment

<details>
<summary>$ rome rage --summary</summary>

```
Environment Variables
  {COLORTERM: "truecolor", LANG: "en_US.UTF-8", TERM: "xterm-256color", TERM_PROGRAM: "vscode"}

User Config
  unset

Rome Version
  10.0.7-beta.18

Node Version
  14.17.0

Platform
  darwin x64 19.6.0

Terminal Features
  {
    background: "unknown"
    colorDepth: 24
    cursor: true
    hyperlinks: true
    isTTY: true
    progress: true
    unicode: true
    columns: OneIndexedNumber<140>
  }

Client Flags
  {
    clientName: "cli"
    silent: false
    cwd: AbsoluteFilePath</Users/Yasser/Dev/rome>
    realCwd: AbsoluteFilePath</Users/Yasser/Dev/rome>
  }

Server Status
  {
    projects: [{id: 0}, {id: 1}]
    workers: [{astCacheSize: 0, heapTotal: 46_514_176, ownedBytes: 0n, ownedFileCount: 0, pid: 10_357, uptime: 0.574993275}]
    server: {heapTotal: 46_514_176, pid: 10_357, uptime: 0.575145362}
  }
```
</details>

## Steps To Reproduce

1. Create a file `test.ts` containing:
```js
switch (true) {
  // test
  // comments
}
```
2. `rome parse test.ts` shows that the comments are not attached to a node.
```js
JSRoot {
  body: [
    JSSwitchStatement {
      cases: []
      discriminant: JSBooleanLiteral {value: true}
    }
  ]
  comments: [CommentLine {id: "0", value: " test"}, CommentLine {id: "1", value: " comments"}]
  corrupt: false
  diagnostics: []
  directives: []
  hasHoistedVars: false
  sourceType: "module"
  syntax: ["ts"]
  integrity: {hash: "731e0783c1b502aa80416d75264e3856f1b9cf5c764bd8b2016e0cd2e1ea6c07"}
  path: UIDPath<project-rome/test.ts>
}
```
3. `rome check test.ts --format-only` erases the comments.
```js
switch (true) {
}
``` 

## Expected Results

Inner comments of a switch statement should not be erased when formatting.

**Comment by Xiaohanxu1996** [comment]
I will take a look on this issue

**Comment by jamiebuilds** [comment]
Hi! Thank you for opening this issue

We recently made the decision to [migrate the Rome codebase to Rust](https://rome.tools/blog/2021/09/21/rome-will-be-rewritten-in-rust) and making some significant changes to its architecture. We've also decided not to release the previous version written in TypeScript.

We're going through and closing pull requests and issues that were specific to the TypeScript codebase, including this one. Thank you for taking the time to open it.

Thanks again!
